crates/server/src/store/blocklist.rs

```diff
@@ -74,6 +72,7 @@ pub struct BlocklistAuthority {
     ttl: u32,
     block_message: Option<String>,
     consult_action: BlocklistConsultAction,
+    log_clients: bool,
```

Comment:
Do we really want to make this configurable? It feels to me like `RUST_LOG` already provides enough control to enable or disable logging on a pretty fine-grained basis.